identity/app/views/fragments/profile/privacyForm.scala.html

```diff
@@ -0,0 +1,43 @@
+@(idUrlBuilder: services.IdentityUrlBuilder,
+  idRequest: services.IdentityRequest,
+  user: com.gu.identity.model.User,
+  privacyForm: Form[form.PrivacyFormData])(implicit request: RequestHeader, messages: play.api.i18n.Messages)
+
+@import views.html.fragments.form.{checkboxField}
+@import form.IdFormHelpers._
+@import helper._
+
+
+<form class="form js-public-profile-form" novalidate action="@idUrlBuilder.buildUrl("/privacy/edit", idRequest)" role="main" method="post">
```

Comment:
Ok I understand. Keep it consistent for now, we can fix it properly later. Thanks.